src/components/Bar.jsx

```diff
@@ -237,7 +237,10 @@ export class Bar extends Component {
           {barLeft || this.renderLeft()}
           {barCenter || this.renderCenter()}
           <div className="u-flex-grow">
-            {barSearch || (searchBarEnabled ? <SearchBar /> : null)}
+            {barSearch ||
+              (searchBarEnabled ? (
+                <SearchBar client={this.props.cozyClient} />
```

Comment:
What is the gain to pass client as props? Why not using `withClient` as the rest of our codebase?

Reply:
I will use `useClient()` , it will be more straightforward.

Reply:
fixed